node/common/transport/mqtt/lib/mqtt.d.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { Packet } from 'mqtt';
+import { X509, results } from 'azure-iot-common';
+import MqttReceiver = require('./mqtt_receiver');
+
+declare class Mqtt {
+    // There is currently no way for the parameter to be a whole TypeScript namespace
```

Comment:
Mqtt.js exposes multiple functions and classes in it's interface, so in Typescript the best way to describe it would be through a namespace or module declaration. But since this is a parameter for a constructor I don't think Typescript has a way to describe that phenomena of requiring a module as a parameter.